draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -860,6 +862,10 @@ cookie        = cookie-string
 cookie-string = cookie-pair *( ";" SP cookie-pair )
 ~~~
 
+Servers MUST be tolerant to potentially receiving multiple cookie
+headers. For example, if an HTTP/2 {{RFC9113}} or HTTP/3 {{RFC9114}} connection
+splits a cookie header to improve compression.
```

Comment:
Nit: The following reads slightly better to me.

```suggestion
headers. For example, an HTTP/2 {{RFC9113}} or HTTP/3 {{RFC9114}} connection might
split a cookie header to improve compression.
```

Reply:
Done